.github/workflows/no-response.yml

```diff
@@ -0,0 +1,33 @@
+# A workflow to close issues where the author hasn't responded to a request for
+# more information; see https://github.com/godofredoc/no-response for docs.
+
+name: response check
+
+# Both `issue_comment` and `scheduled` event types are required.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
```

Comment:
This seems excessive. Once per day? I dunno what it costs us...

Reply:
Good point/  I cribbed this entirely but on reflection, I agree.  I expect it's a cheap workflow but still.

@devoncarew:  curious what you think?

Reply:
Here's daily:
   
    '0 0 * * *'